airflow/api_fastapi/views/public/connections.py

```diff
@@ -45,3 +46,21 @@ async def delete_connection(
         raise HTTPException(404, f"The Connection with connection_id: `{connection_id}` was not found")
 
     session.delete(connection)
+
+
+@connections_router.get(
+    "/connections/{connection_id}",
+    status_code=200,
```

Comment:
I think the 200 code is handled automatically by fastapi

Reply:
Indeed, we can get rid of this line, by default `200` is returned